components/admin_console/group_settings/groups_list/groups_list.tsx

```diff
@@ -268,16 +298,16 @@ export default class GroupsList extends React.PureComponent {
         return newSearchString.replace(/\s{2,}/g, ' ');
     }
 
-    handleFilterCheck = (updates) => {
+    public handleFilterCheck(updates: any[]): void {
```

Comment:
Any chance we could type this? :)

Reply:
typed using tuple [string,boolean], works ?